apps/src/dance/ai/utils/calculateOutputSummedWeights.ts

```diff
@@ -6,22 +6,29 @@ import {CachedWeightsMapping} from '../types';
  * @param {*} emojis, set of three emojis selected by user
  * @param {*} outputWeightsMapping, precalculated association weights for a possible type of output
  *    (e.g. background effect, foreground effect or palette)
- * @returns 1d array of values between [0, 3] with indexes that map to output "option" (e.g. "circles"
- *    for BackgroundEffects) Higher values = stronger correlation
+ * @returns 2d array of values between [0, 3] with an associated output "option"
+ *    (e.g. [1.2, "circles"] for BackgroundEffects).
```

Comment:
You could add:
'The values represent the correlation between the 3 emojis (summed weights) and each output "option".'

Reply:
I like that! Will update.